From 2e0c50f604f15bcb14e1771932d44142b49c311a Mon Sep 17 00:00:00 2001
From: Phi-Bang Nguyen <pnguyen@baylibre.com>
Date: Fri, 30 Apr 2021 21:49:55 +0200
Subject: [PATCH 5/5] libcamera: pipeline: simple: Add support for Mediatek
 i300 platform

The Mediatek i300 platform uses the same SENINF driver as the i500
but it has a converter device with "lower version" named MDP, which
can be used to generate up to three streams.

Upstream-Status: Pending

Signed-off-by: Phi-Bang Nguyen <pnguyen@baylibre.com>
Signed-off-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
(cherry picked from commit 194c38e2e23a1b89a149e55752dd6a0483e51776)
Signed-off-by: Vitor Sato Eschholz <vsatoes@baylibre.com>
---
 src/libcamera/pipeline/simple/simple.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libcamera/pipeline/simple/simple.cpp b/src/libcamera/pipeline/simple/simple.cpp
index de9b2da7..0a2b2ea2 100644
--- a/src/libcamera/pipeline/simple/simple.cpp
+++ b/src/libcamera/pipeline/simple/simple.cpp
@@ -194,7 +194,7 @@ static const SimplePipelineInfo supportedDevices[] = {
 	{ "imx7-csi", { { "pxp", 1 } } },
 	{ "j721e-csi2rx", {} },
 	{ "mxc-isi", {} },
-	{ "mtk-seninf", { { "mtk-mdp3", 3 } } },
+	{ "mtk-seninf", { { "mtk-mdp", 3 }, { "mtk-mdp3", 3 } } },
 	{ "qcom-camss", {} },
 	{ "sun6i-csi", {} },
 };
-- 
2.25.1

